src/org/opensha/refFaultParamDb/vo/FaultSectionPrefData.java

```diff
@@ -638,10 +638,18 @@ public Element toXMLMetadata(Element root, String name) {
 	@SuppressWarnings("unchecked")
 	public static FaultSectionPrefData fromXMLMetadata(Element el) {
 
+		
 		int sectionId = Integer.parseInt(el.attributeValue("sectionId"));
 		String sectionName = el.attributeValue("sectionName");
 		String shortName = el.attributeValue("shortName");
 		double aveLongTermSlipRate = Double.parseDouble(el.attributeValue("aveLongTermSlipRate"));
+		
+		// Wrap this in an exception handler 
+		// as we have seen XML files with this attribute named incorrectly as 'slipRateStDev'
+		double slipRateStdDev;
+		try { slipRateStdDev = Double.parseDouble(el.attributeValue("slipRateStdDev"));}
+		catch (NullPointerException err) {slipRateStdDev = Double.NaN;}
```

Comment:
I would not silently let this pass. If you've added this exception handling because of an input error, it's better to tell the user! Otherwise we're not doing what the user thinks is happening

Reply:
@voj Kevin has merged it as-is. I sort of agree, perhaps I should have added a console message. I didn't want to blow up as StdDev isn't really used except by us & I have  added integration tests to make sure it's working .

 I'm going to drop this PR as it's no longer required...

Reply:
👍